testUtils/ingressCatalogStateMocks.ts

```diff
@@ -0,0 +1,15 @@
+import { Constants } from 'shared';
+import { appCatalogsResponse } from 'testUtils/mockHttpCalls';
+
+const ingressCatalog = appCatalogsResponse.find(
+  (c) => c.metadata.name === Constants.INSTALL_INGRESS_TAB_APP_CATALOG_NAME
+);
+export const catalogsState = {
```

Comment:
Needing to specify this state for the tests seem dirty to me, but I couldn't bring the tests to execute the neccessary catalog loading

Reply:
Don't think there's another way, as the component requires the catalog to be already in the store before loading the apps for it